### PR TITLE
Fix AdminPolicy spec: set the roles correctly before testing the policy

### PR DESCRIPTION
**Pull request by weedySeaDragon**
PT Story:  https://www.pivotaltracker.com/story/show/146118515

Changes proposed in this pull request:
1.  fix the spec so that the correct roles are actually tested

Ready for review:
@patmbolger  @RobertCram

### CODE REVIEW

File: spec/policies/admin_policy_spec.rb [hunk 1]
```diff
@@ -15,30 +15,30 @@
 
   describe 'For a member that is a part of a company' do
     let(:members_company) { Company.find_by_company_number('5562728336')}
```
Comment (patmbolger) [inline spec/policies/admin_policy_spec.rb]: This statement is not needed.

Comment (weedySeaDragon) [reply spec/policies/admin_policy_spec.rb]: oops -- the next line should use`members_company`:

```
subject { described_class.new(members_company).authorized? }
``` 

good catch!

File: spec/policies/admin_policy_spec.rb [hunk 2]
```diff
@@ -15,30 +15,30 @@
...
-    subject { described_class.new(admin).authorized? }
+    subject { described_class.new(members_company).authorized? }
```
Comment (patmbolger) [inline spec/policies/admin_policy_spec.rb]: Not sure I follow this .... the intent of this test is to check auth for a member.  Shouldn't the statement then be:
```
subject { described_class.new(members).authorized? }
```
The way it is now, we're passing a `company` to the auth method (?)

File: spec/policies/admin_policy_spec.rb [hunk 3]
```diff
@@ -15,30 +15,30 @@
...
 
-    it { is_expected.to be_truthy }
+    it { is_expected.to be_falsey }
 
   end
 
   describe 'For a member that is not part of a company' do
-    subject { described_class.new(admin).authorized? }
+    subject { described_class.new(member).authorized? }
```
Comment (patmbolger) [inline spec/policies/admin_policy_spec.rb]: In order to be a `member`, doesn't there have to be a company associated with that member?  Perhaps this test should be for a user who has applied for membership but not yet accepted, using factory ```:user_with_membership_app```?

**Review by patmbolger — COMMENTED** [review]
Please see comments.

File: spec/policies/admin_policy_spec.rb (hunk 2 above)
Comment (weedySeaDragon) [reply spec/policies/admin_policy_spec.rb]: On further review, you're absolutely right.
This doesn't make any sense.  Checking a _member_ and a _user_ is what is needed; a _company_ makes no sense.  (perhaps I was just thinking of a member associated with a different company.  Who knows.)

**Review by patmbolger — APPROVED** [review]

**Pull request merged**